syntax = "proto3";

package bakery;

message NewMacaroonRequest {
  string identifier = 1; // The identifier is the public part of the macaroons.
  repeated string first_party_caveats = 2;
}

message MacaroonSerialized {
    string serialized = 1;
}

message ValidMacaroon {
  bool valid = 1;
}

service Bakery {
  rpc CreateMacaroon (NewMacaroonRequest) returns (MacaroonSerialized);
}


